.github/workflows/checker-framework.yml

```diff
@@ -12,14 +12,14 @@ jobs:
       fail-fast: false
       matrix:
         profile:
-          - checker-framework-nullness-optional-interning
-          - checker-framework-methods-resource-fenum
-          - checker-framework-lock-tainting
-          - checker-framework-index
-          - checker-framework-formatter
-          - checker-framework-signature-gui-units-init
-          - checker-framework-regex-property-key-compiler-message
-          - checker-framework-purity-value-returns
+          - checker-nullness-optional-interning
+          - checker-methods-resource-fenum
+          - checker-lock-tainting
```

Comment:
You need to change the `id` of profiles too, you can find the profiles here:
https://github.com/checkstyle/checkstyle/blob/a3224dafe53f501d3ea91f4eb3bd6e30e3c8e4b2/pom.xml#L2404-L2407

Reply:
I see. So it was included for all those files that includes "checker-framewor". Right?

Reply:
Yes, all such names are from pom.xml